vuework02/src/views/Step2.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div id="app">
+    <div class="worksheet">
+	<div class="title">
+      <span class="step">STEP2</span>
+      <p id="title">以下にお答えください</p>
+	</div>
+	<div>
+		<p>現在、生命保険に加入していますか？</p>
```

Comment:
インデントが空きすぎていますので修正しましょう。
インデントは半角スペース２つです。
他の部分も同様です。
```suggestion
  <p>現在、生命保険に加入していますか？</p>
```